pkg/oc/cli/importimage/importimage.go

```diff
@@ -159,6 +169,16 @@ func (o *ImportImageOptions) Validate(cmd *cobra.Command) error {
 	return nil
 }
 
+// Validate ensures that a ImportImageOptions is valid and can be used to execute
+// an import.
+func (o *ImportImageOptions) Validate(cmd *cobra.Command) error {
+	if len(o.Target) == 0 {
+		return kcmdutil.UsageErrorf(cmd, "you must specify the name of an image stream")
```

Comment:
`Validate` only works with `*Options` and does not accept any input, return `fmt.Errorf` here.